server/env.ts

```diff
@@ -0,0 +1,19 @@
+// Throws on bad tokens
+export function getJwtSecret(): string {
```

Comment:
this is a good spot for these helper functions, thanks for cleaning up the salt getter! :)